subsys/net/lib/openthread/platform/uart.c

```diff
@@ -92,17 +91,12 @@ static void uart_rx_handle(void)
 static void uart_tx_handle(void)
 {
 	uint32_t len;
-	const uint8_t *data;
-
-	len = ring_buf_get_claim(ot_uart.tx_ringbuf, (uint8_t **)&data,
-				 ot_uart.tx_ringbuf->size);
-	if (len) {
-		int err;
 
-		len = uart_fifo_fill(ot_uart.dev, data, len);
-		err = ring_buf_get_finish(ot_uart.tx_ringbuf, len);
-		(void)err;
-		__ASSERT_NO_MSG(err == 0);
+	if (s_write_length) {
```

Comment:
It won't compile like this, you missed to update any variable references as well.

Reply:
Yes, sorry, I did commit without saving file. Fixed.